frontend/src/components/broadcast/owl.css

```diff
@@ -0,0 +1,149 @@
+.carousel-div {
+  padding: 10px 0px;
+  margin-left: 180px;
+  margin-right: 180px;
+}
+.Bottom_bgbox_first {
+  height: 100px;
+  background-color: #fafafa;
+}
+.Bottom_bgbox {
+  height: 365px;
+  background-color: #016795;
+  margin: 40px 0px 120px 0px;
+}
+.owl-nav button {
+  position: absolute;
+  background-color: #1b2431;
+  color: #1b2431;
+  margin: 0;
+  transition: all 0.3s ease-in-out;
+  width: 50px;
+  height: 50px;
+  border-radius: 80px;
+}
+.owl-nav button.owl-prev {
+  background-color: #1b2431 !important;
+  color: #fff !important;
+  padding: 0 15px !important;
+  left: -10%;
+}
+
+.owl-nav button.owl-next {
+  background-color: #1b2431 !important;
+  color: #fff !important;
+  padding: 0 15px !important;
+  right: -10%;
+}
+
+.owl-dots {
+  text-align: center;
+  padding-top: 15px;
+}
+.owl-dots button.owl-dot {
+  width: 5px;
+  height: 5px;
+  border-radius: 50%;
+  display: inline-block;
+  background: #ccc;
+  margin: 0 3px;
+}
+.owl-dots button.owl-dot.active {
+  background-color: #000;
+}
+.owl-dots button.owl-dot:focus {
+  outline: none;
+}
+.owl-dots button.owl-dot span {
+  display: none !important;
+}
+.owl-nav button {
+  position: absolute;
+  top: 40%;
+  transform: translateY(-50%);
+  background: rgba(255, 255, 255, 0.38) !important;
+  width: 50px;
+  height: 50px;
+  border-radius: 80px;
+}
+span {
+  font-size: 30px;
+  position: relative;
+  top: -5px;
+}
+.owl-nav button:focus {
+  outline: none;
+}
+
+.modal-window {
+  position: fixed;
+  background-color: rgba(255, 255, 255, 0.25);
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  visibility: hidden;
+  opacity: 0;
+  pointer-events: none;
+  transition: all 0.3s;
+  &:target {
+    visibility: visible;
+    opacity: 1;
+    pointer-events: auto;
+  }
+  & > div {
+    width: 400px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    padding: 2em;
+    background: #ffffff;
+  }
+  header {
+    font-weight: bold;
+  }
+  h1 {
+    font-size: 150%;
+    margin: 0 0 15px;
+  }
+}
+
+.modal-close {
+  color: #aaa;
+  line-height: 50px;
+  font-size: 80%;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 0;
+  width: 70px;
+  text-decoration: none;
+  &:hover {
+    color: black;
+  }
+}
+
+/* For mobile phones: */
+[class*="col-"] {
+  width: 100%;
+}
```

Comment:
This piece of code will affect all the **bootstrap col** classes across the pages. Please remove it. 

You may refer to [this](http://owlcarousel2.github.io/OwlCarousel2/demos/responsive.html) to make owl carousel responsive.

Reply:
ok

Reply:
can you tell me the dimensions of mobile view??

Reply:
You will get the dimensions in DevTools.

Reply:
I am done please check my pr